lib/SPIRV/libSPIRV/SPIRVDecorate.h

```diff
@@ -500,6 +515,20 @@ class SPIRVMemberDecorateMergeINTELAttr : public SPIRVMemberDecorate {
   }
 };
 
+class SPIRVMemberDecorateBankBitsINTELAttr : public SPIRVMemberDecorate {
+public:
+  // Complete constructor for BankBitsINTEL decoration
+  SPIRVMemberDecorateBankBitsINTELAttr(SPIRVEntry *TheTarget,
+                                       SPIRVWord MemberNumber,
+                                       const std::vector<SPIRVWord> &TheBits)
+      : SPIRVMemberDecorate(DecorationBankBitsINTEL, MemberNumber, TheTarget) {
+    for (auto &I : TheBits) {
```

Comment:
Just a nit:
Literals.insert(Literals.end(), TheBits.begin(), TheBits.end());

Reply:
Why we can't just call the copy constructor?
```suggestion
    Literals(TheBits) {
```

Reply:
Copy constructor here can be called as `Literals = TheBits`. Seems, that it's the simplest way to add elements to Literals.